packages/docs/base/icons.md

```diff
@@ -0,0 +1,206 @@
+# Iconography
```

Comment:
@joejezowski-okta @arnoldsandoval-okta @johnintrater-okta "Icons" or "Iconography" when talking about the guidelines generally?

Reply:
Not sure how valid this source is, but they have an interesting take on the two terms:
https://wikidiff.com/iconography/icon#:~:text=As%20nouns%20the%20difference%20between,an%20object%20of%20religious%20devotion.

If this is accurate, I would say iconography works better. I also like how it associates itself to "typography".

Reply:
Swapping to Icons for now, we can always adjust later.

Reply:
Bah! Didn't refresh and missed this comment! 

Shall I swap both to -ography in that case?